services/chainstorage/src/main/java/tech/pegasys/teku/services/chainstorage/EphemeryDatabaseReset.java

```diff
@@ -38,8 +37,13 @@ Database resetDatabaseAndCreate(
       } else {
         slashProtectionDir = validatorDataDir.resolve("slashprotection");
       }
+
+      System.out.println("beaconDir" + beaconDataDir);
+      System.out.println("networkFile" + networkFile);
+      System.out.println("dbdir" + dbDataDir);
```

Comment:
we can't ever merge System printlns...

Reply:
Apologies, forgot to remove it before pushing.